### PR TITLE
Adding Maryland Interstates

### PR DESCRIPTION
**Pull request by talllguy**
This is my first attempt at a PR on this repo. I included MD IMAP data for interstate highways. There are other centerlines but I wanted to make sure this is right before proceeding.

### CODE REVIEW

File: sources/us/md/md-interstates.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "coverage": {
+        "country": "us",
+        "state": "md",
+        "city": "Statewide",
```
Comment (iandees) [inline sources/us/md/md-interstates.json]: You can ditch the "city" key when it's statewide.

File: sources/us/md/md-interstates.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "US Census": {
+            "geoid": "24"
+        },
+        "geometry": {
```
Comment (iandees) [inline sources/us/md/md-interstates.json:8]: You can ditch the geometry because we'll be able to draw the state border.

**Comment by iandees** [comment]
This looks good. I just added some nitpicky comments.

**Comment by talllguy** [comment]
@iandees thanks. This is fun! I dropped the city line.

**Comment by talllguy** [comment]
@iandees can you pull, or should I?

File: sources/us/md/md-interstates.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            ]
+        }
+    },
+    "data": "http://data.imap.maryland.gov/datasets/82842b1ff67b466086a4c097c6e7aa46_0",
+    "website": "http://data.imap.maryland.gov/datasets/82842b1ff67b466086a4c097c6e7aa46_0.geojson",
+    "type": "geojson",
+    "conform": {
+        "type": "geojson",
+        "name": "ROADNAMELO"
+        }
```
Comment (iandees) [inline sources/us/md/md-interstates.json]: I think this `}` is extra.

**Comment by iandees** [comment]
Woot! Thanks!

**Pull request merged**